internal/pkg/cli/svc_deploy.go

```diff
@@ -51,6 +51,10 @@ const (
 	fmtForceUpdateSvcComplete = "Forced an update for service %s from environment %s.\n"
 )
 
+var aliasUsedWithoutDomainFriendlyTest = fmt.Sprintf("To use %s, your application must be associated with a domain: %s.\n",
```

Comment:
nit: did you mean
```suggestion
var aliasUsedWithoutDomainFriendlyText = fmt.Sprintf("To use %s, your application must be associated with a domain: %s.\n",
```
??

Reply:
😳